6/체육복.js

```diff
@@ -0,0 +1,32 @@
+function solution(n, lost, reserve) {
+  let gymUniformCount = 0;
+  const students = [];
+
+  for (let gymUniform = 0; gymUniform < n; gymUniform++) {
```

Comment:
```suggestion
  for (let gymUniform = 0; gymUniform < n; gymUniform++) {
```
for문에서는 index 약자로 i 사용 가능합니다!